helm/Chart.yaml

```diff
@@ -7,9 +7,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.11.0
+version: 0.13.0
```

Comment:
Could we set this to a single minor version bump?

Reply:
I set this to 0.12.0 in both this PR and in the #13 PR as well so both changes need to be in together prior to deploying the 0.12.0 chart version